__tests__/Uncontrolled.js

```diff
@@ -1 +1,113 @@
-test('please, please test me', () => {})
+import React from 'react'
+import axe from 'axe-core'
+import { act, fireEvent, render } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import Zoom from '../source'
+
+const testA11y = html =>
+  new Promise((resolve, reject) => {
+    axe.run(html, {}, (err, { violations }) => {
+      if (err) {
+        reject(err)
+      }
+      if (violations.length > 0) {
+        reject(violations)
+      }
+      resolve(true)
+    })
+  })
+
+test('is accessible with defaults & <img />', async () => {
```

Comment:
the a11y tests use promises, and they unfortunately do not work well at all with the `jest.useFakeTimers()`, so i've split them out